platforms/azure/modules/lbaas/lbaas.tf

```diff
@@ -91,17 +98,17 @@ resource "azurerm_lb_probe" "lb" {
 }
 
 output "vip_address" {
-  value = "${azurerm_public_ip.lb.ip_address}"
+  value = "${data.azurerm_public_ip.lb.*.ip_address}"
 }
 output "vip_type" {
   value = "A"
 }
 
 locals {
-  pool_ids = "${compact(concat(azurerm_lb_backend_address_pool.lb.*.id,list("")))}"
+  pool_ids = "${compact(concat(azurerm_lb_backend_address_pool.lb.*.id,list("")))}" 
```

Comment:
????

Reply:
Oh, extra whitespace. This should be removed of course.